lookout/style/typos/cmdline_tools.py

```diff
@@ -0,0 +1,291 @@
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "-trs", "--train_size", required=False, type=int,
+        default=50000,
+        help="Train dataset size.")
+
+
+def add_test_size_arg(my_parser: ArgumentParser):
+    """
+    Add a test size argument to an argparse parser.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "-ts", "--test_size", required=False, type=int,
+        default=10000,
+        help="Test dataset size.")
+
+
+def add_save_train_path_arg(my_parser: ArgumentParser):
+    """
+    Add a save train path argument to an argparse parser.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "-strp", "--save_train_path", required=False, type=str,
+        help="Path to save train dataset to.")
+
+
+def add_save_test_path_arg(my_parser: ArgumentParser):
+    """
+    Add a save test path argument to an argparse parser.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "-stp", "--save_test_path", required=False, type=str,
+        help="Path to save test dataset to.")
+
+
+def add_threads_number_arg(my_parser: ArgumentParser):
+    """
+    Add a threads number argument to an argparse parser.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "-tn", "--threads_number", required=False, type=int, default=8,
+        help="Number of threads for multiprocessing.")
+
+
+def add_save_model_path_arg(my_parser: ArgumentParser):
+    """
+    Add a save model path argument to an argparse parser.
+
+    :param my_parser: Parser to add the argument to.
+    """
+    my_parser.add_argument(
+        "-smp", "--save_model_path", required=False, type=str,
+        default="lookout/style/typos/data/new_corrector.asdf",
+        help="Path to save model to.")
+
+
+def add_fasttext_path_arg(my_parser: ArgumentParser, addition: str = ""):
+    """
+    Add a fasttext path argument to an argparse parser.
+
+    :param my_parser: Parser to add the argument to.
+    :param addition: Addition to the default help message.
+    """
+    my_parser.add_argument(
+        "-ft", "--fasttext_path", required=False, type=str,
+        help="Path to a FastText model dump." + addition)
+
+
+def create_parser() -> ArgumentParser:
+    """
+    Create a parser for the lookout.style.typos utility.
+
+    :return: an ArgumentParser with an handler defined in the handler attribute.
+    """
+    # Deferred imports to speed up loading __init__
+    from lookout.style.typos.train_test_corrector import (get_train_test, prepare_data, test,
+                                                          train, train_from_scratch)
+
+    parser = ArgumentParser(formatter_class=ArgumentDefaultsHelpFormatterNoNone)
+
+    # General options
+    parser.add("--log-level", default="DEBUG", help="Log verbosity level.")
```

Comment:
Can you please use `lookout.core.cmdline.add_logging_args` here and in `lookout.style.format.cmdline_tools`